scripts/install-dependencies.sh

```diff
@@ -1,35 +1,9 @@
 #!/usr/bin/env bash
 
-# This script does `yarn install` if a `yarn.lock` file is present, otherwise `npm install`
-
 set -e
 
-if [[ -n "${FAIL_ON_DIRTY_LOCKFILE}" ]]; then
-  if [[ "${YARN_VERSION:0:2}" == "2." ]]; then
-    YARN_OPTS="--immutable"
-  else
-    YARN_OPTS="--frozen-lockfile"
-  fi
-  NPM_CMD="ci"
-else
-  YARN_OPTS=""
-  NPM_CMD="install"
-fi
+# Execute install with corepack, return true to ignore errors from non matching package managers
```

Comment:
Any way we could allow users to provide args/flags for these commands?

Reply:
Do you mean to corepack or the package manager? Or both.

Reply:
The package manager commands. My concern is if a user wants to pass some args to their package manager of choice, how do we enable that?

Not sure if Docker's ENV would work to pass args down, or if there's another way.

e.g. pseudocode

```
corepack yarn install $OPTS || true
corepack pnpm install $OPTS || true
corepack npm ci $OPTS       || true
```